react/features/base/session/constants.js

```diff
@@ -0,0 +1,12 @@
+
+export const SESSION_CONFIGURED = Symbol('SESSION_CONFIGURED');
+
+export const SESSION_ENDED = Symbol('SESSION_ENDED');
+
+export const SESSION_FAILED = Symbol('SESSION_FAILED');
```

Comment:
Generally one wants to handle ending and failure the same way, with maybe something extra in case of failure. IMHO it would be better to have a single state SESSION_ENDED, with some metadata attached to it. Some `error` object for example, which would be `null` iff there was no error.

Reply:
I'm open to that. Lets wait for @lyubomir